src/pages/cw-multi-test/getting-started/counter.mdx

```diff
@@ -4,4 +4,4 @@ tags: ["multitest", "getting started", "counter"]
 
 # Counter
 
-(tbd)
+see: https://github.com/CosmWasm/docs/pull/169
```

Comment:
For future. please just rebase PR onto previous one

Reply:
Ok.